src/com/carolinarollergirls/scoreboard/ScoreBoardManager.java

```diff
@@ -118,41 +111,9 @@ private static void loadVersion() {
         printMessage("Carolina Rollergirls Scoreboard version "+getVersion());
     }
 
-    private static void loadProperties() {
```

Comment:
This leaves `FileInputStream`, `FileNotFoundException`, `HashMap`, `Map`, and `ConcurrentHashMap` unused.